src/server/_params.py

```diff
@@ -477,6 +480,8 @@ def parse_time_set() -> TimeSet:
         # old version
         require_all("time_type", "time_values")
         time_values = extract_dates("time_values")
+        if len(time_values) == 1 and time_values[0] == "*":
```

Comment:
is this something you tried?
```suggestion
        if time_values == ["*"]:
```

demo:
```
>>> ["*"] == ["*"]
True
>>> ["*"] == "*"
False
>>> ["abc"] == ["*"]
False
>>> ["*","abc"] == ["*"]
False
>>> [] == ["*"]
False
```